### PR TITLE
Parsing description as markdown

### LINKED ISSUE #2: Parse Podspec.description as Markdown
Even in the simple example we have someone using markdown in the long description.

### PR DESCRIPTION
**Pull request by marcelofabri**
Fixes #2

### CODE REVIEW

File: lib/cocoapods_acknowledgements/plist_generator.rb [hunk 1]
```diff
@@ -1,7 +1,15 @@
+require 'redcarpet'
+
 module CocoaPodsAcknowledgements
   class PlistGenerator
     class << self
 
+      @@markdown_parser = Redcarpet::Markdown.new(Redcarpet::Render::HTML, extensions = {})
+
+      def markdown_parser
+        @@markdown_parser
```
Comment (segiddins) [inline lib/cocoapods_acknowledgements/plist_generator.rb]: just make this

```ruby
def markdown_parser
  @markdown_parser ||= Redcarpet::Markdown.new(Redcarpet::Render::HTML)
end
```

Comment (marcelofabri) [inline lib/cocoapods_acknowledgements/plist_generator.rb]: Hah, I'm still getting these Ruby stuff. Thanks!

**Comment by segiddins** [comment]
@orta what do you think of also parsing the license text as MD?

**Comment by orta** [comment]
I don't think we need it for the license - I think this is totally awesome :+1:

**Pull request merged**